src/Reveal.Sdk.Dom/Visualizations/CustomVisualization.cs

```diff
@@ -6,10 +6,24 @@
 
 namespace Reveal.Sdk.Dom.Visualizations
 {
+    /// <summary>
+    /// The custom visualizion is used to support visualizations not supported out of the box. This type of visualization uses a web view to pass data to the visualization for display.

```

Comment:
Drop "This type of visualization uses a web view to pass data to the visualization for display." This sentence is not applicable within the DOM